integration-tests/tests/package.json

```diff
@@ -20,14 +23,18 @@
     "@types/chai": "^4.1.7",
     "@types/fs-extra": "^9.0.12",
     "@types/mocha": "^5.2.6",
+    "@types/node": "^16.11.12",
     "concurrently": "^6.0.2",
     "fs-extra": "^7.0.1",
     "mocha": "^9.1.3",
     "node-fetch": "^2.6.1",
+    "platform": "^1.3.6",
     "realm": "*",
-    "realm-app-importer": "*"
+    "realm-app-importer": "*",
+    "rollup": "^2.61.1"
   },
   "dependencies": {
```

Comment:
Aren't these all dev-dependencies?

Reply:
Actually `@thi.ng/bench` can now be moved to the dev dependencies, but the other two are used at runtime.